src/Constants/index.ts

```diff
@@ -3,9 +3,13 @@ import { DeepKeys } from '@tanstack/react-table';
 
 export * from './Options';
 
+// TABLE PAGINATION CONSTANTS
 export const DATA_PER_PAGE = 50;
 export const PAGE_SIZE_OPTIONS = [10, 25, 50, 100, 200];
 
+// TABLE POLLING CONSTANTS
```

Comment:
we can improve this maybe. we are sure the lambda function takes ~30 sec to run... so maybe in the interval [20, 60] seconds we need to poll more frequently. can schedule it better